### ISSUE TITLE
Disable emojione CDN, use custom emojione lib

### ISSUE THREAD

**Issue by mervick** [mervick/emojionearea#116]

**Comment by SirCumz** [comment]
yeah, when you install emojionearea trough NPM it will install emojione package as well version 3, which does not work for the current version of emojionearea.

i think emojionearea should be compatible with emojione version 3+, remove the CDN and mention the dependency in the docs.

**Comment by carpii** [comment]
The latest commits suggest emojionearea now supports EmojiOne v3.x

Maybe this ticket now is invalid? 

@mervick - great job on cleaning up the issue list lately anyway!

**Comment by mervick** [comment]
> Maybe this ticket now is invalid?

It's not yet,
There is no way to disable emojione CDN

**Comment by mervick** [comment]
>  when you install emojionearea trough NPM it will install emojione package as well version 3

NPM by default installs the latest version of `emojione`,
but you can configure it in your local `package.json`.

EmojioneArea supports any version of `emojione` `>=2.0.1`

https://github.com/mervick/emojionearea/blob/dd2a048aff730546cdd7c8c56ca6c0881a992ba8/package.json#L30-L31
